pkg/oc/cli/describe/projectstatus.go

```diff
@@ -1026,7 +1103,36 @@ func describeSourceInPipeline(source *buildapi.BuildSource) (string, bool) {
 	return "", false
 }
 
-func describeDeployments(f formatter, dcNode *appsgraph.DeploymentConfigNode, activeDeployment *kubegraph.ReplicationControllerNode, inactiveDeployments []*kubegraph.ReplicationControllerNode, restartFn func(*kubegraph.ReplicationControllerNode) int32, count int) []string {
+func describeDeployments(f formatter, dNode *kubegraph.DeploymentNode, activeDeployment *kubegraph.ReplicaSetNode, inactiveDeployments []*kubegraph.ReplicaSetNode, restartFn func(node *kubegraph.ReplicaSetNode) int32, count int) []string {
+	if dNode == nil {
+		return nil
+	}
+	out := []string{}
+	deploymentsToPrint := append([]*kubegraph.ReplicaSetNode{}, inactiveDeployments...)
+	revision, _ := deployutil.Revision(dNode.Deployment)
+
+	deploymentsToPrint = append([]*kubegraph.ReplicaSetNode{activeDeployment}, inactiveDeployments...)
+	for i, deployment := range deploymentsToPrint {
+		restartCount := int32(0)
+		if restartFn != nil {
+			restartCount = restartFn(deployment)
+		}
+		out = append(out, describeDeploymentStatus(deployment.ReplicaSet, revision, i == 0, restartCount))
+	}
+	return out
+}
+
+func describeDeploymentStatus(rs *kapisext.ReplicaSet, revision int64, first bool, restartCount int32) string {
+	timeAt := strings.ToLower(formatRelativeTime(rs.CreationTimestamp.Time))
+	replicaSetRevision, _ := deployutil.Revision(rs)
+	if replicaSetRevision == revision {
```

Comment:
@kargakis @tnozicka i really hope there is something better to report the state of RS created/managed by deployment... The only thing that RS contain is revision so I know this RS is "old", but it does not say anything about if it failed/etc.

Reply:
@spadgett since you did the console pages for deployments